docs/examples/userguide/language_basics/exception_example.pyx

```diff
@@ -0,0 +1,113 @@
+# >>> c.a_cpdef_wo_py_retval()
+# Exception: a_cpdef_wo_py_retval: This Exception will cause a return of 0
+# Exception ignored in: 'cython_exception_example.a_cpdef_wo_py_retval'
+# Traceback (most recent call last):
+#   File "<stdin>", line 1, in <module>
+# Exception: a_cpdef_wo_py_retval: This Exception will cause a return of 0
+# 0
+
+cpdef int a_cpdef_wo_py_retval():
+	# NB: defined as cpdef
```

Comment:
> NB: defined as cpdef

Seems unnecessary - it's on the previous line and in the name.

Reply:
I probably overstated the cpdef point because the case was not mentioned in the docs